includes/Core/Util/Migration_1_128_0.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Migration for 1.128.0
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Modules\Modules;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Modules\Analytics_4\Settings as Analytics_Settings;
+use Google\Site_Kit\Modules\Ads;
+use Google\Site_Kit\Modules\Ads\Settings as Ads_Settings;
+
+/**
+ * Class Migration_1_128_0
+ *
+ * @since 1.128.0
+ * @access private
+ * @ignore
+ */
+class Migration_1_128_0 {
+	/**
+	 * Target DB version.
+	 */
+	const DB_VERSION = '1.128.0';
```

Comment:
I think it makes more sense to rename the class name to something like `Migration_Conversion_ID` and then use `n.e.x.t` as the `DB_VERSION`. In this case, it will be clear what this migration is about and the version will be automatically generated when we actually release this migration.

```php
class Migration_Conversion_ID {

	/**
	 * Target DB version.
	 */
	const DB_VERSION = 'n.e.x.t';
```

Reply:
Good idea.